react-native-aztec/ios/RNTAztecView/RCTAztecViewManager.m

```diff
@@ -16,6 +16,9 @@ @interface RCT_EXTERN_MODULE(RCTAztecViewManager, NSObject)
 RCT_EXPORT_VIEW_PROPERTY(onActiveFormatsChange, RCTBubblingEventBlock)
 RCT_EXPORT_VIEW_PROPERTY(onActiveFormatAttributesChange, RCTBubblingEventBlock)
 
+RCT_EXPORT_VIEW_PROPERTY(accessibilityLabel, NSString)
+RCT_EXPORT_VIEW_PROPERTY(accessibilityHint, NSString)
+
```

Comment:
For this to work we should use `RCT_REMAP_VIEW_PROPERTY` instead.
But my question is, do we want to add extra labels and hints to the Aztec TextView?

Personally, I think it's better for the system to handle accessibility on UITextViews since it's quite complex.

As an example, adding the current label and hint, the VoiceOver will read on Aztec's empty post title:

Label: "Title, Empty. TextField, is editing. 0 width space, character mode, insertion point at end. 
Hint: "Update the title. Use the rotor to access misspelled words."

Maybe we are complicating the already complex TextView labels and hint, it might get in the way of the users instead of helping.

Having this label and hint in the wrapper view should be enough to give the user the info needed, then they can just start editing in the TextView after performing the action.

What do you think? @enejb

Reply:
Ok not it makes sense to me on a new post where the title is automatically focused, not having this is confusing. 🤔

Reply:
Yeah I am not sure what the best approach is here. To me it also sounds like lot of text. 

I just wanted to make sure that the user know that they are editing a the post title when they get dropped in creating a new post or page.

Reply:
> For this to work we should use RCT_REMAP_VIEW_PROPERTY instead.

So, this was a mistake on my part:

`RCT_EXPORT_VIEW_PROPERTY` will work just fine, and the property will be set to the view's `accessibilityHint` property as expected.

`RCT_REMAP_VIEW_PROPERTY` is used when the JS name doesn't match the View's property name. This gives an opportunity to re-name the property.

Reply:
I remove the `RCT_EXPORT_VIEW_PROPERTY` since it wasn't doing anything.